.evergreen/install-tools.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Script for installing various tool dependencies.
+
+# variables
+PROJECT_DIRECTORY=${PROJECT_DIRECTORY:-$PWD}
+SWIFT_VERSION=${SWIFT_VERSION:-5.2.4}
+INSTALL_DIR="${PROJECT_DIRECTORY}/opt"
+
+export SWIFTENV_ROOT="${INSTALL_DIR}/swiftenv"
+export PATH="${SWIFTENV_ROOT}/bin:$PATH"
+
+# usage: build_from_gh [name] [url] [tag]
+build_from_gh () {
```

Comment:
we have to build swiftformat and swiftlint ourselves on linux, but sourcery we can just download a binary since it runs on macOS